### PR TITLE
Fail gracefully if nothing to monitor

### LINKED ISSUE #10: Issue with monitor
Hi,

We are getting the following error when running tests... These tests were run before without any issues.
What can be the cause of this, I have tested this in CACHE 2018, Ensemble 2016.2, and IRIS For Health 2021.1, they all have the same issue.

I can not find any results searching for this on the internet.

image

### PR DESCRIPTION
**Pull request by isc-tleavitt**
Fixes #10 , #11

### CODE REVIEW

File: module.xml [hunk 1]
```diff
@@ -2,16 +2,14 @@
...
+  <Version>2.1.3</Version>
   <Description>Run your typical ObjectScript %UnitTest tests and see which lines of your code are executed. Includes Cobertura-style reporting for use in continuous integration tools.</Description>
   <Packaging>module</Packaging>
   <Resources>
     <Resource Name="TestCoverage.PKG">
     </Resource>
     <Resource Name="TestCoverage.DataType.PKG">
     </Resource>
-    <Resource Name="TestCoverage.Utils.LineByLineMonitor.CLS">
-    </Resource>
```
Comment (isc-tedpeck) [inline module.xml:14]: This <Resource> removal seems unrelated to the rest of the changes. Just remarking in case it's not intentional.

**Comment by isc-tleavitt** [comment]
Need to figure out what's going on with CI, but don't need to block on that.
